dist/booking.js

```diff
@@ -23,7 +23,11 @@ const filterToBook = (lessons) => {
         .second(0)
         .isSameOrBefore(moment()) &&
       // Is the class after current date / time
-      moment(date).isAfter(moment())
+      moment(date)
+        .hour(23)
```

Comment:
Why would you set the time of the gym class to the end of the day? I think what we want here is to check if the date / time of the gym class is after current time, `moment(date).isAfter(moment())` would also allow a booking to the same day as long as the time is after the current time no?

Reply:
This function only have visibility to the 'lesson' date, check the following case:

| code | value | alias |
|---|---|---|
| date | 2017-11-30 | |
| moment(date) | 2017-11-30 00:00:00 | |
| moment() | 2017-11-30 09:30:00 | |
| moment(date).hour(23).minute(59).second(59) | 2017-11-30 23:59:59 | class_time_EOD |
| moment(date).isAfter(moment()) | false | |
| class_time_EOD.isSameOrAfter(moment()) | true |

that commit is quick fix, but works well

Reply:
Ok 👍